music/README.md

```diff
@@ -8,19 +8,19 @@ For the Python TensorFlow implementations, see the [main Magenta repo](https://g
 
 ## Contents
 
-* [Example Applcations](#example-applications)
-* [Supported Models](#supported-models)
-* [Getting Started](#getting-started)
-* [Model Checkpoints](#model-checkpoints)
+- [Example Applications](#example-applications)
+- [Supported Models](#supported-models)
+- [Getting Started](#getting-started)
+- [Model Checkpoints](#model-checkpoints)
 
 ## Example Applications
 
 Here are a few applications built with MagentaMusic.js:
 
-* [Beat Blender](https://g.co/beatblender) by [Google Creative Lab](https://github.com/googlecreativelab)
```

Comment:
@cghawthorne I had `prettify` left on which fixed a bunch of whitespace and stuff, let me know if you want me to undo this

Reply:
I think it's fine.